packages/broker/src/ConfigWizard.ts

```diff
@@ -0,0 +1,195 @@
+    },
+    apiAuthentication: null,
+    plugins: {}
+}
+
+export class ConfigWizard{
+    config: Config
+    // required to mock results on tests
+    inquirer: inquirer.Inquirer = inquirer
+
+    defaultWebsocketPort = 7171 
+    defaultMqttPort = 7272 
+    defaultHttpPort = 7373
+
+    constructor(){
+        this.config = DefaultConfig
+    }
+
+    async inquirerSinglePrompt(prompt: inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion) {
+        const answers = await this.inquirer.prompt([prompt])
+        return answers[prompt.name!]
+    }
+
+    async generateOrImportPrivateKey(): Promise<string>{
+        const generateOrImport = await this.inquirerSinglePrompt({
+            type: 'list',
+            name:'generateOrImportEthereumPrivateKey',
+            message: 'Do you want to generate a new Ethereum private key or import an existing one?',
+            choices: ['Generate', 'Import'],
+            filter: (choice: string) => {
+                return choice.toLowerCase()
+            }
+        })
+
+        if (generateOrImport === 'generate'){
+            this.config.ethereumPrivateKey = Wallet.createRandom().privateKey
+        } else if (generateOrImport === 'import'){
+            const privateKey = await this.inquirerSinglePrompt({
+                type: 'input',
+                name: 'privateKey',
+                message: "'Please provide the private key to import'",
+            })
+
+            try {
+                const wallet = new Wallet(privateKey)
+                this.config.ethereumPrivateKey = wallet.privateKey
+            } catch (privateKeyError){
+                throw new Error(`Invalid privateKey provided for import: ${privateKey}`)
+            }
+
+        } else {
+            throw new Error(`Invalid option ${generateOrImport} provided`)
+        }
+
+        return this.config.ethereumPrivateKey
+    }
+
+    async promptNumberWithDefault(prompt: inquirer.InputQuestion, defaultValue: number){
+        const valueString = await this.inquirerSinglePrompt(prompt)
+        let value = parseInt(valueString)
+        if (valueString === ''){
+            value = defaultValue
+        }
+
+        if (!Number.isNaN(value) && Number.isInteger(value) && value > 1023 && value < 49151){
+            return value
+        } else {
+            return defaultValue
+        }
+    }
+
+    async selectPlugins(): Promise<Array<string>>{
+        const plugins: inquirer.Answers = await this.inquirerSinglePrompt( {
+            type: 'checkbox',
+            name:'selectedItems',
+            message: 'Select the plugins to enable',
+            choices: [
+                {name: 'Websocket'},
+                {name:'MQTT'},
+                {name:'HttpPublish'}
+            ]
+        })
+
+        const selectedPlugins = []
+        for (let i = 0; i < plugins.length; i++){
+            selectedPlugins.push(plugins[i])
+            if (plugins[i] === 'Websocket'){
```

Comment:
You might want to configure your IDE to add whitespace after if/else if/else-statements for consistent styling with rest of code base.
```suggestion
            if (plugins[i] === 'Websocket') {
```